### PR TITLE
adding support for PDOK NL

### PR DESCRIPTION
**Pull request by opensourceame**
This PR adds support for [pdok](https://www.pdok.nl), a new Geocoding service for The Netherlands that replaces Nationaal Georegister.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@ rdoc/*
 Gemfile.lock
 api_keys.yml
 test/geocoder_test.sqlite
+.idea/*
```
Comment (opensourceame) [inline .gitignore:9]: `.idea` is created by RubyMine which is a very popular IDE

**Comment by opensourceame** [comment]
@alexreisner the old Nationaal Georegister has been deprecated. I can remove the code from the old platform in this PR or a create a new PR for that. What would you prefer?

File: .gitignore (hunk 1 above)
Comment (alexreisner) [reply .gitignore:9]: Yes, however, everything in this `.gitignore` file should be specific to the Geocoder gem. To ignore files related to local development, please use your `~/.gitignore_global` file.

**Comment by opensourceame** [comment]
> Thanks for this! It looks great. If you can remove that line from `.gitignore`, I'll merge. And feel free to submit another request for removing the deprecated API.

Done @alexreisner 👍

**Pull request merged**